src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+public class Ball {
+
```

Comment:
[Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery)
Please don’t add redundant empty lines to your code.